### PR TITLE
[js] fix library bundling and some trivial improvement

### PR DESCRIPTION
**Pull request by fs-eire**
**Description**: fix library bundling and some trivial improvement for JS.

- updated webpack config. now onnxruntime-web can be consumed correctly as NPM package.
   - applied "browser" field in package.json so filter out non-browser environment dependencies: `fs`, `path` and `util`.
   - rewrite `buildConfig()` function so it generate correct config for both ort{}.js and ort-web{}.js
   - set umd property for external 'onnxruntime-common' (ort-web{}.js)
   - setup fallback to off for ort{}.js and ort-web{}.js, but enable node polifill for test runner
- update license link in README (NPM page requires an absolute link to make it work)
- add missing copyright header
- sort items in package.json
- add keyword in package.json

This change depends on #7535 and #7549. should have a merge after these 2 PR is completed.

### CODE REVIEW

File: js/web/webpack.config.js [hunk 1]
```diff
@@ -1,90 +1,85 @@
...
+  config.resolve.fallback = { path: false, fs: false, util: false };
+  return config;
+}
+
+// "ort-web{.min|.node}.js" config
+function buildOrtWebConfig({
   suffix = '',
   format = 'umd',
-  target = 'ES2017',
+  target = 'es5',
```
Comment (Tixxx) [inline js/web/webpack.config.js:59]: can we just make this es6 by default? I see most of the configs below are using es6.

Comment (fs-eire) [reply js/web/webpack.config.js:59]: they are different targets. for compatibility concerns, we use es5 by default; for es6, we add ".es6" in the file name, so they are separated files

**Review by Tixxx — APPROVED** [review]

File: js/web/webpack.config.js [hunk 2]
```diff
@@ -139,10 +134,26 @@ module.exports = () => {
...
+      // ort.es6.min.js
+      buildOrtConfig({ suffix: '.es6.min', target: 'es6' }),
+      // ort.es6.js
+      buildOrtConfig({ suffix: '.es6', mode: 'development', devtool: 'inline-source-map', target: 'es6' }),
+
+      // ort-web.min.js
+      buildOrtWebConfig({ suffix: '.min' }),
+      // ort-web.js
+      buildOrtWebConfig({ mode: 'development', devtool: 'inline-source-map' }),
+      // ort-web.min.js
```
Comment (fs-eire) [inline js/web/webpack.config.js:150]: should be ort-web.es6.min.js

File: js/web/webpack.config.js [hunk 3]
```diff
@@ -139,10 +134,26 @@ module.exports = () => {
...
+      buildOrtWebConfig({ suffix: '.es6.min', target: 'es6' }),
+      // ort-web.js
```
Comment (fs-eire) [inline js/web/webpack.config.js:152]: should be ort-web.es6.js

**Review by hanbitmyths — DISMISSED** [review]

**Review by hanbitmyths — APPROVED** [review]

**Pull request merged**